e2e/internal/upgrade/upgrade_test.go

```diff
@@ -125,6 +124,20 @@ func TestUpgrade(t *testing.T) {
 	testNodesEventuallyHaveVersion(t, k, targetVersions, *wantControl+*wantWorker, *timeout)
 }
 
+func runUpgradeCheck(require *require.Assertions, cli string) {
+	cmd := exec.CommandContext(context.Background(), cli, "upgrade", "check", "--debug")
+	msg, err := cmd.CombinedOutput()
+	require.NoErrorf(err, "%s", string(msg))
```

Comment:
Why did you choose `require.NoErrorf` over `require.NoError` ?

Reply:
can't remember. seems unnecessary. changed.